noxfile.py

```diff
@@ -180,3 +181,64 @@ def make(session: nox.Session):
         *(args.make_args or ("clean", "coredocs")),
     ]
     session.run("make", "-C", "docs/docsite", *make_args, external=True)
+
+
+def _rm_tmpdir(tmpdir):
+    rmtree(tmpdir, ignore_errors=True)
```

Comment:
Handling the removal like this is not necessary. nox sets TMPDIR to a custom directory and automatically clears it before each run.

Reply:
Yes, but wouldn't that still leave an extra clone on the filesystem? AFAIK nox doesn't offer a way to clean the virtual environment after the run.

Reply:
I tend to agree with @oraNod here that leaving the clones hanging out because nox will remove before the next run isn't the way to go. 

It's not how I'd expect a tool to behave, which is why I did it this way.